frontend/src/app/AppRouter.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
+
+import ProtectedRoute from "./ProtectedRoute";
+
+import LoginPage from "../login/LoginPage";
+import HomePage from "../homepage/HomePage";
+import DeviceListPage from "../devicelist/DeviceListPage";
+import DeviceDetailsPage from "../deviceDetailsPage/DeviceDetailsPage";
+import StreamListPage from "../streamlist/StreamListPage";
+import CreateStreamPage from "../createStream/CreateStreamPage";
+import LogListPage from "../loglist/LogListPage";
+import PathNotFoundPage from "../general/PathNotFoundPage";
+
+export default function AppRouter() {
+  return (
+    <BrowserRouter>
+      <Switch>
+        <ProtectedRoute path="/Login" render={() => <LoginPage />} />
+        <ProtectedRoute
+          path="/Home"
```

Comment:
nah, no real point in letting users change the path names

Reply:
what users? i meant like the wya we have `localhost` and stuff as vars in `.env.development` for ourselves if we need to change path names in the future
to avoid stuff like the big stream path rename

Reply:
The reason urls like localhost are in environment variables is that we change them in different environments. Path names won't change by environment so if we do extract them we can put them in a constants file. But imo that would belong in a separate PR like the big stream path rename was.